README.md

```diff
@@ -232,35 +271,30 @@ As Device Farm supports additional test types the plugin must be extended to ena
   1. If the test type requires a test artifact package to be uploaded in addition to the app then make sure the new class extends the TestPackageProvider trait.
   2. If the test type supports the 'filter' parameter extend the HasFilter trait.
 3. Modify DeviceFarmExtension to add a method for the new test type, like so:
-```
-    void mynewtesttype(final Closure closure) {
-        NewTestType newTest = new NewTestType()
-        project.configure newTest, closure
-        test = newTest
-    }        
+```java
+void mynewtesttype(final Closure closure) {
+    NewTestType newTest = new NewTestType()
+    project.configure newTest, closure
```

Comment:
We're missing parentheses around the arguments here.

Reply:
Nice find! I accidentally tagged the annotation as java rather than groovy. It's been updated in the latest commit.